mediorum/ddl/ddl.go

```diff
@@ -25,29 +30,41 @@ var mediorumMigrationTable = `
 
 func Migrate(db *sql.DB) {
 	mustExec(db, mediorumMigrationTable)
+
+	// TODO: safe to remove after everyone runs it once
+	slog.Info("checking if it's time to re-seed...")
+	var reseeding bool
+	db.QueryRow(`select count(*) = 1 from mediorum_migrations where hash = 'reseeding'`).Scan(&reseeding)
+	if reseeding {
+		slog.Info("re-seeding and restarting server...")
```

Comment:
when are the nodes going to restart a second time to run the actual reseeding after the reseed has been scheduled on the first restart / iteration of `Migrate`?